examples/TheoJansenStrandbeest.cpp

```diff
@@ -415,7 +415,7 @@ void UserInputHandler::handleEvent(State& state, Real accuracy,
     }  
 }
 
-
+/*
```

Comment:
You could convert the whole comment to the `/* */` kind if that would get rid of the warning. That would be an improvement, I think.